### PR TITLE
New feature to set environment variables as arguments.

### PR DESCRIPTION
**Pull request by Galironfydar**
Hi, based on my issue I submitted, I'd thought i'd give it a go. Hopefully i've done everything correctly. This is my first PR.

Many Thanks.

### CODE REVIEW

**Comment by Galironfydar** [comment]
Closes #11

File: src/DockerContainer.php [hunk 1]
```diff
@@ -86,6 +91,13 @@ public function mapPort(int $portOnHost, $portOnDocker): self
         return $this;
     }
 
+    public function setVariable(string $envName, string $envValue): self
```
Comment (freekmurze) [inline src/DockerContainer.php:94]: Let's make it very explicit and rename this to `setEnvironmentVariable`

Comment (Galironfydar) [reply src/DockerContainer.php:94]: That's fine by me! I did think about calling it that.

**Comment by freekmurze** [comment]
Perfect, thanks!

**Comment by Galironfydar** [comment]
Thanks for adding 😄

**Pull request merged**